docs/auto-qc/iqms_intepretability.md

```diff
@@ -0,0 +1,1288 @@
+# Interpretability of the Image Quality Metrics (IQMs) of MRIQC
+
+[MRIQC](https://mriqc.readthedocs.io/en/latest/) is a powerful tool to assess the quality of MR images in a research study. In addition to a visual report, a number of image quality metrics (IQMs) is generated. However, there is a large number of these metrics and it is not immediately obvious which IQM a researcher should pay most attention to when deciding over the quality of a given image.
```

Comment:
Can I ask to break lines after period (for readability):
```suggestion
[MRIQC](https://mriqc.readthedocs.io/en/latest/) is a powerful tool to assess the quality of MR images in a research study.
In addition to a visual report, a number of image quality metrics (IQMs) are generated.
However, there is a large number of these metrics and it is not immediately obvious which IQM a researcher should pay most attention to when deciding over the quality of a given image.
```

(also, fixed a typo)